### DISCUSSION TITLE
Some suggestions, more in line with the Win11 style.

### DISCUSSION THREAD

**Discussion by gtazoo** [valinet/ExplorerPatcher#1824] [Ideas]
![屏幕截图 2023-08-11 175447](https://github.com/valinet/ExplorerPatcher/assets/135148751/963544d4-7e86-4100-ace7-00d521c6c955)
![屏幕截图 2023-08-11 175308](https://github.com/valinet/ExplorerPatcher/assets/135148751/da454a62-f4b8-4246-b6d1-b0410dd5dc68)
It is recommended to change these to rounded corners, just like the style of Start11, which is more in line with the win11 style.

**Comment by pyrates999** [comment]
What setting are you referring to?

**Reply by gtazoo** [reply]
Options on the left, such as the shutdown button, are right angles in ExplorerPatcher. I mean it's possible to change these flat designs to mica effects or rounded corners and add switch options where possible

**Reply by pyrates999** [reply]
No, I mean what option in explorer patcher are you changing that would need this?

**Reply by gtazoo** [reply]
![屏幕截图 2023-08-11 210733](https://github.com/valinet/ExplorerPatcher/assets/135148751/57b1ede8-597c-4c93-b947-85dcba4af2d1)
Suggest questions
![屏幕截图 2023-08-11 212143 png](https://github.com/valinet/ExplorerPatcher/assets/135148751/3a159f3d-d140-44e5-9b61-d0553272d1d2)

Design reference scenarios

**Comment by gtazoo** [comment]
These change ideas, is the workload too large
